Userland/Libraries/LibWeb/HTML/ShadowRealmGlobalScope.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024, Shannon Booth <shannon@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/DOM/EventTarget.h>
+
+namespace Web::HTML {
+
+// https://whatpr.org/html/9893/webappapis.html#shadowrealmglobalscope
+class ShadowRealmGlobalScope : public DOM::EventTarget {
```

Comment:
This should inherit from Bindings::ShadowRealmGlobalScopeGlobalMixin, which should be generated for all `[Global]` interfaces.

Reply:
Right, I had left a FIXME on this one in the IDL file, but not in the C++ source/header file. Yup, will need to do that - this will involve a bit of reshuffling of the mixins we have currently.